.eslintrc.json

```diff
@@ -55,6 +62,7 @@
     "no-named-as-default": "off",
     "arrow-body-style": ["error", "as-needed"],
     "import/no-dynamic-require": "off",
+    "prettier/prettier":0,
```

Comment:
I think it wouldn't be a good idea to disable our code formatter: prettier unless there is a technical reason why. If any please state it.

Reply:
What?
DId I do that? let me remove it

Reply:
Thanks for pointing that out @Musigwa, I'm not disabling the code formatted, 
I used that rule to disable some useless warning I was getting from hound CI
Let me share a screenshot about that error.

Reply:
<img width="773" alt="Screen Shot 2019-05-13 at 14 44 04" src="https://user-images.githubusercontent.com/18573157/57622362-b6400c00-758d-11e9-8e59-bd73b571cda4.png">